var/graphs/mbta/router-config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "stopClusterMode": "parentStation",
+  "useFlexService": "true",
+  "updaters": [
+    {
+      "type": "real-time-alerts-enhanced",
+      "frequencySec": 100,
+      "url": "https://cdn.mbta.com/realtime/Alerts_enhanced.json",
+      "feedId": "1"
+    },
+    {
+      "type": "stop-time-updater",
+      "frequencySec": 1000,
+      "sourceType": "gtfs-http",
+      "url": "https://cdn.mbta.com/realtime/TripUpdates!.pb",
```

Comment:
no `!`

Reply:
Sorry, my bad, this is the config I was using for testing. Fixed.